ibm_catalog.json

```diff
@@ -0,0 +1,145 @@
+          }
+        ],
+        "flavors": [
+          {
+            "label": "Fully configurable",
+            "name": "fully-configurable",
+            "install_type": "fullstack",
+            "working_directory": "solutions/fully-configurable",
+            "architecture": {
+              "descriptions": "This architecture deploys Red Hat OpenShift Cluster on VPC enhanced with AI, accelerating machine learning and AI workload deployments. It outlines the necessary infrastructure components for AI-powered applications and provides a streamlined setup for data science and AI development.",
+              "features": [
+                {
+                  "title": "Create Resource Group",
+                  "description": "No"
+                },
+                {
+                  "title": "Use Existing Resource Group",
+                  "description": "Yes"
+                },
+                {
+                  "title": "Create Red Hat OpenShift Cluster",
+                  "description": "Yes"
+                },
+                {
+                  "title": "Use Existing Red Hat Openshift Cluster",
+                  "description": "No"
+                }
+              ],
+              "diagrams": [
+                {
+                  "diagram": {
+                    "caption": "Red Hat OpenShift cluster with AI for IBM Cloud.",
+                    "url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-ocp-ai/blob/main/reference-architecture/deployable-architecture-ocp-ai-cluster.svg",
+                    "type": "image/svg+xml"
+                  },
+                  "description": "This architecture illustrates the deployment of Red Hat OpenShift cluster integrated with an AI, showcasing the key components and their interactions for AI workload support."
+                }
+              ]
+            },
+            "configuration": [
+              {
+                "key": "default_worker_pool_machine_type"
+              },
+              {
+                "key": "default_worker_pool_workers_per_zone"
+              },
+              {
+                "key": "default_worker_pool_operating_system"
+              },
+              {
+                "key": "default_gpu_worker_pool_storage"
+              },
+              {
+                "key": "additional_worker_pools"
+              }
+            ],
+            "dependencies": [
+              {
+                "name": "add-ons-beta-deploy-arch-ibm-base-ocp-vpc",
+                "catalog_id": "49e71b32-78af-46b4-9869-388c17fff716",
```

Comment:
This needs to be switched to the community registry catalog ID (which is `7a4d68b4-cf8b-40cd-a3d1-f49aff526eb3`) once its published there